verl/utils/dataset/rl_dataset.py

```diff
@@ -399,19 +565,38 @@ def __getitem__(self, item):
             convert_video_to_images = not processor_supports_video(self.processor)
         
         messages = self._build_messages(row_dict, convert_video_to_images=convert_video_to_images)
+        if "audio" in self.modalities:
+            # NOTE: Set the following prompt for qwen omni when we are training on audio
+            messages.insert(0, {
+                "role": "system",
+                "content": [
+                    {"type": "text", "text": "You are Qwen, a virtual human developed by the Qwen Team, Alibaba Group, "
+                                             "capable of perceiving auditory and visual inputs, as well as generating text and speech."}
+                ]
+            })
         model_inputs = {}
+        
+        # NOTE: DEBUGGING
+        dbg = False
+        if dbg:
+            print(f"[getitem] idx=? ds={row_dict.get('dataset')} src={row_dict.get('data_source')} "
+                f"modalities={self.modalities}")
 
         if self.processor is not None:
+            # THIS CHUNK IS BASICALLY ABOUT PROCESSING ALL THE MODALITIES
             from verl.utils.dataset.vision_utils import process_image, process_video
+            from verl.utils.dataset.audio_utils import process_audio
+            
+            if dbg:
+                print(f"[prompt] raw_prompt_chars={len(raw_prompt)}")
 
             raw_prompt = self.processor.apply_chat_template(
                 messages, add_generation_prompt=True, tokenize=False, **self.apply_chat_template_kwargs
             )
             multi_modal_data = {}
```

Comment:
I think self.apply_chat_template_kwargs is passed here. Is there a place where it's not passed?